model/AI/Classes/EnergyConsumption

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumption
+
+## Summary
+
+Class that contains properties to describe EnergyConsumption incurred by a AI model in different stages of its lifecycle.
+
+## Description
+
+The class used for denoting the training energy consumption, inference energy consumption and fine tuning energy consumption of the AI model(s) used in a AI software.
```

Comment:
Using "system" instead of "software"?

"used in a AI software" -> "used in an AI system"